crates/sui-source-validation/src/tests.rs

```diff
@@ -329,12 +342,8 @@ async fn module_not_found_on_chain() -> anyhow::Result<()> {
         panic!("Expected verification to fail");
     };
 
-    let SourceVerificationError::OnChainDependencyNotFound { package, module } = err else {
-        panic!("Expected OnChainDependencyNotFound, got: {:?}", err);
-    };
-
-    assert_eq!(package, "b".into());
-    assert_eq!(module, "c".into());
+    let expected = expect!["On-chain version of dependency b::c was not found."];
+    expected.assert_eq(&err.to_string());
```

Comment:
Here and elsewhere I simplified cases where the expected value implies the assertions/expected error types.